openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/H2OXgboostUtils.java

```diff
@@ -38,11 +38,7 @@
      * The set of parameters that are possible to define during the creation of an H2O XGBoost model.
      */
     public static final Set<ModelParameter> PARAMETERS =
-            ParametersBuilderUtil.getParametersFor(XGBoostParametersV3.class, water.bindings.pojos.XGBoostParametersV3.class)
-                    .stream()
-                    // This is the same as "ntrees" so it is redundant to have 2 mandatory fields that are the same.
-                    .filter(modelParam -> !"n_estimators".equals(modelParam.getName()))
-                    .collect(Collectors.toSet());
+            ParametersBuilderUtil.getParametersFor(XGBoostParametersV3.class, water.bindings.pojos.XGBoostParametersV3.class);
```

Comment:
This is no longer needed because the field is now being removed because it's not in the XGBoostParametersV3 "fields" field.